pkg/command/service/load.go

```diff
@@ -305,8 +295,78 @@ func getReplicasCount(loadResult pkg.LoadResult) (int, []int) {
 	return maxReplicasCount, replicasCountList
 }
 
+// runInternalVegeta runs internal load test tool(vegeta) using library, returns load output and error
+func runInternalVegeta(inputs pkg.LoadArgs, endpoint string, host string) (output string, err error) {
+	concurrency, err := strconv.ParseUint(inputs.LoadConcurrency, 10, 64)
+	if err != nil {
+		return "", errors.New(fmt.Sprintf("failed to get load concurrency: %s", err))
```

Comment:
Better to use fmt.Errorf(...) instead of errors.New(fmt.Sprintf(...))

Reply:
Thank you, fixed.